Sources/MongoSwift/MongoCursor.swift

```diff
@@ -16,13 +18,19 @@ public class MongoCursor<T: Codable>: Sequence, IteratorProtocol {
      *   - `UserError.invalidArgumentError` if the options passed to the command that generated this cursor formed an
      *     invalid combination.
      */
-    internal init(fromCursor cursor: OpaquePointer,
-                  withClient client: MongoClient,
-                  withDecoder decoder: BSONDecoder) throws {
+    internal init(from cursor: OpaquePointer,
```

Comment:
while we're here and changing stuff, can you add a typealias for the cursor pointer?